Acronyms (#4)
alphagov/tech-docs-linter: CHANGELOG.md

# Changelog

## v0.2.0

### New features

> - Add rule to enforce that acronyms are defined on first use. An initial list of exceptions has been agreed by the technical writers

Reviewer: ```suggestion
- Add rule to enforce that acronyms are defined on first use. GDS technical writers have agreed an initial list of exceptions
```

Reviewer: Just a minor suggestion to make it more active
